packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -100,6 +100,19 @@ class ObjCSelector {
   final String value;
 }
 
+/// Metadata to annotate methods to control the signature used for Swift output.
+/// The number of components in the provided signature must match the number of
```

Comment:
The first sentence should be in it's own paragraph, according to our style guidelines. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-specific-requirements

Reply:
Fixed